packages/remix-cloudflare-workers/worker.ts

```diff
@@ -36,15 +42,15 @@ export function createRequestHandler({
 }) {
   let handleRequest = createRemixRequestHandler(build, mode);
 
-  return (event: FetchEvent) => {
+  return (event: FetchEvent | Evt) => {
     let loadContext = getLoadContext?.(event);
 
     return handleRequest(event.request, loadContext);
   };
 }
 
 export async function handleAsset(
-  event: FetchEvent,
+  event: FetchEvent | Evt,
   build: ServerBuild,
   options?: Partial<KvAssetHandlerOptions>
 ) {
```

Comment:
Right below this, you need to pass [`ASSET_NAMESPACE` and `ASSET_MANIFEST`](https://github.com/cloudflare/kv-asset-handler#asset_namespace-required-for-es-modules) to `getAssetFromKV`. Otherwise Cloudflare won’t be able to bind the KV to the worker.